src/GovUk.Education.ExploreEducationStatistics.Data.Seed/Services/SeedService.cs

```diff
@@ -22,7 +22,7 @@ public class SeedService : ISeedService
         private readonly IMapper _mapper;
         private readonly string _storageConnectionString;
         private readonly IFileStorageService _fileStorageService;
-        
+
```

Comment:
@SiShakes are you happy if we delete the Seed project now?

Reply:
Yes I think it can be retired now.